src/modules/home/components/header/constants/guest-types.constant.ts

```diff
@@ -0,0 +1,30 @@
+export const guestTypes = [
+    {
+        type: 'adults',
+        title: 'Adults',
+        description: 'Ages 13 or above',
```

Comment:
(Optional) I think 13- to 19-year-olds are called teenagers, and 20+ is adults. So how about adding another type here to separate them?

Reply:
I think adding more guest types would be unnecessary right now because we have nothing to use them in db or backend. Can we do this later?

Reply:
Yeah, sure. It's optional anyway.